analysis/config_global.yaml

```diff
@@ -8,5 +8,9 @@ source:
   prepare_data: source/prepare_data
   descriptive: source/descriptive
 
+# External Direcotries to record
+external:
+  self: ..
```

Comment:
I'm not sure this is how we want to demonstrate this feature. I was imagining adding this to `paper_slides` and then have it work on `input`.